routes/services/study/getStudyList.ts

```
@@ -3,6 +3,77 @@ import { RequestHandler } from "express";
+        break;
+      case "point_desc":
+        orderByCondition.point = "desc";
+        break;
+      case "date_asc":
+        orderByCondition.created_at = "asc";
+        break;
+      case "date_desc":
+      default:
+        orderByCondition.created_at = "desc";
+        break;
+    }
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)